libqtile/window.py

```diff
@@ -197,24 +165,45 @@ def __init__(self, window, qtile):
         }
         self.update_hints()
 
-    x = property(fset=_geometry_setter("x"), fget=_geometry_getter("x"))
-    y = property(fset=_geometry_setter("y"), fget=_geometry_getter("y"))
+    @property
+    def x(self):
```

Comment:
Can we make these function calls? I know qtile uses properties a lot, but I find it quite hard to follow, and I'd like to avoid more of it.

Reply:
FWIW it's not "more" of it since these were properties already. I'd rather not change the API here, although I do agree that we should probably pay more attention to the properties we add in the future.

Reply:
Ah, yes, I missed that they were properties before. So ignore me :)